package.json

```diff
@@ -4,7 +4,11 @@
   "author": "iExec",
   "license": "Apache-2.0",
   "scripts": {
-    "test-builds": "bash test-builds.sh"
+    "test-builds": "bash test-builds.sh",
+    "codegen": "graph codegen subgraph.template.yaml",
+    "test": "npm run codegen && graph test",
+    "test-docker": "npm run codegen && docker run -it --rm -v $(pwd):/matchstick/subgraph rainprotocol/matchstick:main",
```

Comment:
why having test docker ?

Reply:
Because `test` command does not work natively on some platforms (https://github.com/graphprotocol/graph-tooling/blob/main/packages/cli/src/commands/test.ts#L230)